dll/win32/shell32/wine/shellpath.c

```diff
@@ -2128,8 +2158,46 @@ HRESULT WINAPI SHGetFolderPathAndSubDirW(
         hr = E_FAIL;
         goto end;
     }
-
     TRACE("Created missing system directory %s\n", debugstr_w(szBuildPath));
```

Comment:
May move that trace at the very end (just before the "end:" label) ?

Reply:
Excuse me?

Reply:
Don't move. OK.